docker/web/Dockerfile

```
@@ -23,7 +23,7 @@ RUN buildDeps=" \
     && tar xvjf phantomjs-2.1.1-linux-x86_64.tar.bz2 \
     && mv phantomjs-2.1.1-linux-x86_64 /usr/local/bin \
     && ln -sf /usr/local/bin/phantomjs-2.1.1-linux-x86_64/bin/phantomjs /usr/local/bin \
-    && phantomjs --version \
+    && export OPENSSL_CONF=/etc/ssl/ && phantomjs --version \
```

This will work on build, but won't on run (e when you are going to try and export a gantt chart it will fail because environment variable will be undefined)

```suggestion
    && phantomjs --version \
```

and add after line 5 to 

```
ENV OPENSSL_CONF /etc/ssl/
```

---

I've force pushed the correction. I've notice that "Export gantt" didn't work (error 500) on this PR. Maybe fixed in #302 or we have to fix it.